release.sh

```diff
@@ -59,7 +59,7 @@ git commit -a -m "Update changelog for Rally release $RELEASE_VERSION"
 
 # * Update version in `setup.py` and `docs/conf.py`
 echo "Updating release version number"
-echo "$RELEASE_VERSION" > version.txt
+echo "__version__ = \"$RELEASE_VERSION\"" > esrally/_version.py
```

Comment:
nit: to help readability and avoid backslashes one can also leverage `printf`:

`printf '__version__ = "%s"\n' $RELEASE_VERSION > esrally/_version.py`

Reply:
Thanks, fixed